### PR TITLE
TC_02.07.007 | <Homepage> Verify the "Learn more about distributed builds" link

### PR DESCRIPTION
**Pull request by julian-ole**
…lds link

https://trello.com/c/rVlvAGhz/910-at0207007-homepage-verify-the-learn-more-about-distributed-builds-link

### CODE REVIEW

**Comment by KatyaKaterina33** [comment]
Hello, please connect TC and AT cards.

**Comment by 7108581** [comment]
1. Rename your file to  HomepageMainPanel'sHeadingsAndLinks.cy.js
2. Put into describe "HomepageMainPanel'sHeadingsAndLinks" 
Check this link please
https://redroverschool.slack.com/archives/C049HHDG0KF/p1684259040891579?thread_ts=1684249643.909019&cid=C049HHDG0KF

**Comment by 7108581** [comment]
Also, you can press the button "Edit" near the PR name and past the full name of your PR

File: cypress/e2e/homepageHelpLink.cy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  it('AT_02.07.007 | <Main Panel> Verify the "Learn more about distributed builds" link', function () {
+    cy.get(".content-block__help-link")
+      .should("be.visible")
+      .should("have.text", "Learn more about distributed builds")
+      .should("have.attr", "target", "_blank")
+      .invoke("removeAttr", "target")
+      .click();
+    cy.url().should(
+      "eq",
+      "https://wiki.jenkins.io/display/JENKINS/Distributed+builds"
```
Comment (SvetlanaVet) [inline cypress/e2e/homepageHelpLink.cy.js:13]: I'm not sure but maybe better to move "https://wiki.jenkins.io/display/JENKINS/Distributed+builds" in the fixtures and the same about "Learn more about distributed builds"

**Review by SvetlanaVet — APPROVED** [review]

**Pull request merged**